examples/sagemaker-tensorflow/README.md

```diff
@@ -0,0 +1,44 @@
+# Training and Serving Merlin on AWS SageMaker
+
+With AWS Sagemaker, you can package your own models that can then be trained
+and deployed in the SageMaker environment.
+The following notebook will show you how to use NVIDIA Merlin for training and
+inference in the SageMaker environment.
+
+- [Training and Serving Merlin on AWS SageMaker](sagemaker-merlin-tensorflow.ipynb)
+
+It assumes that readers are familiar wtth some basic concepts in NVIDIA Merlin,
+such as:
+
+- Using NVTabular to GPU-accelerate preprocessing and feature engineering,
```

Comment:
My $0.02 is to replace the commas with periods and remove the "and" from the second one. These are perfectly good statements.

If you want to get the gold star in tech writing, you'd change the last bullet for parallelism with the first two.  The first two use nominalized forms of verbs, Using, Training.  Maybe the last could be "Performing inference..."  Totally up to you and whether you want that gold star.

Reply:
I want that ⭐.